index.bs

```diff
@@ -729,13 +812,18 @@ in a [=/Realm=] |realm|, perform the following steps:
 
          Note: It is expected that this atomically updates the contents of the file on disk
          being written to.
+
+      1. [=file entry/lock/release|Release the lock=] on |stream|.[=FileSystemWritableFileStream/[[file]]=].
       1. [=/Resolve=] |closeResult| with `undefined`.
```

Comment:
Just wondering why close() returns undefined instead of success (or failure in the case of an error (close() *can* fail)

Reply:
Yeah, I think this is ok